MagnetScannerIOS/NetworkResolver.swift

```diff
@@ -15,11 +15,11 @@ import SwiftyJSON
 //
 public class NetworkResolver {
   static let API_END_POINT: String = "https://tengam.org/content/v1/search/beacons/"
-  static let RADIUS: Int = 50
+  static let RADIUS: Int = 100
 
   static func resolveLocation(lat: Double, lon: Double, callback: (Array<JSON> -> Void)!) {
```

Comment:
Well, actually it does ask if there are items in this location, it doesn't resolve items. I think resolveLocation is a good name, we could go for findItems if you want.

BTW, not part of this patch :P